src/components/Modals/Receive/ReceiveInfo.tsx

```diff
@@ -81,18 +85,16 @@ export const ReceiveInfo = ({ asset, accountId }: ReceivePropsType) => {
     asset,
     wallet,
     chainAdapter,
-    utxoParams,
+    bip44Params,
   ])
 
   const handleVerify = async () => {
-    const accountParams = utxoParams
-
     if (!(wallet && chainAdapter && receiveAddress)) return
```

Comment:
same deal here, we want to guard for the existence of an `accountType` for UTXO chains before doing a `getAddress` call (paranoia, but to be safe)